include/zephyr/pm/policy.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
  * microseconds. SYS_FOREVER_US value lifts the latency constraint. Other values
  * are forbidden.
  */
-typedef void (*pm_policy_latency_changed_cb_t)(int32_t latency);
+typedef void (*pm_policy_latency_changed_cb_t)(uint32_t latency);
```

Comment:
The general definition of this API makes me uncomfortable. The values are now `uint32_t`, but the documented return value is defined as a negative number? The @param docs also don't really make sense as a result of this change. All return values are positive (since its a `uint32_t`), so there is no space for `SYS_FOREVER_US` and "Other values" are impossible.